EasyPost/Models/Shared/Collection.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using EasyPost._base;
+using Newtonsoft.Json;
+
+namespace EasyPost.Models.Shared
+{
+    public class Collection : EasyPostObject
+    {
+        #region JSON Properties
+
+        [JsonProperty("has_more")]
+        public bool has_more { get; set; }
```

Comment:
Question: What happens if a collection in the future for whatever reason doesn't contain the `has_more` key, will everything blow up?

Reply:
No, it will fallback to `false` by default.

Reply:
That's right, le sigh.